### DISCUSSION TITLE
GItHub Education approved but not available

### DISCUSSION THREAD

**Discussion by CarlosManuel02** [community/community#132443] [GitHub Education]
I applied for the GitHub student pack on 14.07.2024. My application was approved, but it is not yet available.

**Comment by davevad93** [comment]
Hi @CarlosManuel02 ,

Check out the solution in this [discussion](https://github.com/orgs/community/discussions/132236#discussioncomment-10051207) and see if it works for you as well.

**Comment by RajeevRanjan5642** [comment]
Hey👋

I think I can help you with that because I was also in the same boat. I had applied on 29th of June and received mail of confirmation on 2nd of July. They take 4-5 days for processing the request since the appliers are in great number. So, you should just wait for 1-2 days and you will definitely receive a mail from their side 💌.

Hope it helps 💖!!
